include/class.client.php

```diff
@@ -41,12 +41,14 @@ function __call($name, $args) {
         $tag =  substr($name, 3);
         switch (strtolower($tag)) {
             case 'ticket_link':
+                $qstr = array();
+                if ($cfg && $cfg->isAuthTokenEnabled()
+                        && ($ticket=$this->getTicket()))
+                    $qstr['auth'] = $ticket->getAuthToken($this);
+
```

Comment:
I think this can be moved to `::getVar($tag)`, which would simplify the code and perhaps remove the dependency on the __call trampoline